programs/ssl/ssl_mail_client.c

```diff
@@ -57,8 +57,10 @@
 #include <unistd.h>
 #else
 #include <io.h>
-#define read _read
-#define write _write
+#define read(fd, buf, len)                                      \
```

Comment:
personally, I think the MACRO checks should be different, to make it more readable(although this is not part of this PR):
```
#if defined (_MSC_VER)
#else
#endif
```